dotnet/xml/ns-Microsoft.ML.SamplesUtils.xml

```diff
@@ -1,6 +1,8 @@
 <Namespace Name="Microsoft.ML.SamplesUtils">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>
+      Namespace containing utilities to download and featurize datasets for the API reference samples.
+      It is part of the Microsoft.ML.SampleUtils nuget package.
```

Comment:
Should there be a nuget link here?

Reply:
there is no link to it yet, in nuget.org, as it has not been pushed there. We'll edit after it gets pushed. 

---
In reply to: [262682041](https://github.com/dotnet/ml-api-docs/pull/76#discussion_r262682041) [](ancestors = 262682041)